samples/Samples.Elasticsearch/Program.cs

```diff
@@ -238,7 +255,7 @@ private static List<Func<object>> CatCommands(ElasticClient elastic)
                 () => elastic.CatRepositories(new CatRepositoriesRequest()),
                 () => elastic.CatSegments(new CatSegmentsRequest()),
                 () => elastic.CatShards(new CatShardsRequest()),
-                () => elastic.CatSnapshots(new CatSnapshotsRequest()),
+                // () => elastic.CatSnapshots(new CatSnapshotsRequest()),
```

Comment:
Why are all the `CatSnapshot` calls commented? An explanatory comment would be useful for context.

Reply:
Added comments. I don't know that I care enough to look into this one specific API call right now.